src/components/sections/ProductGallery/ProductGalleryPage.tsx

```diff
@@ -49,7 +85,30 @@ function GalleryPage({ page, display, title, fallbackData }: Props) {
         pageSize={itemsPerPage}
         title={title}
       />
-      <ProductGrid products={products} page={page} pageSize={itemsPerPage} />
+      {/* If showSponsoredProducts is on, a ProductTiles will be display in between two blocks of ProductGrid on the page 0 */}
+      {showSponsoredProducts &&
+      page === 0 &&
+      productsSponsored !== undefined &&
+      productsSponsored.length > 1 ? (
```

Comment:
I think adding a const to explicitly says what this conditional is supposed to check would increase readability. Something like `shouldDisplaySponsoredProducts`

Reply:
Agreed, thanks Dan!